code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -104,14 +104,14 @@
 	. = 1
 
 /datum/reagent/drug/crank/addiction_act_stage3(mob/living/M)
-	M.adjustBruteLoss(5*REM, 0)
+	M.adjustInternalLoss(3*REM, 0)
 	..()
 	. = 1
 
 /datum/reagent/drug/crank/addiction_act_stage4(mob/living/M)
 	M.adjustBrainLoss(3*REM)
 	M.adjustToxLoss(5*REM, 0)
-	M.adjustBruteLoss(5*REM, 0)
+	M.adjustInternalLoss(3*REM, 0)
```

Comment:
A crank addiction nerf that I don't believe you mentioned in the description of the PR.

It definitely needed a nerf, but this alone doesn't bring this PR into a positive light for me.

Reply:
lower number doesn't mean nerf if you read the full line

undoc'd tho